codec-stomp/src/test/java/io/netty/handler/codec/stomp/StompTestConstants.java

```diff
@@ -18,8 +18,8 @@
 public final class StompTestConstants {
     public static final String CONNECT_FRAME =
         "CONNECT\n" +
-            "host:stomp.github.org\n" +
             "accept-version:1.1,1.2\n" +
+            "host:stomp.github.org\n" +
```

Comment:
@trustin that depends on the `Map` implementation. With TreeMap we lose insertion order (but not for a header with multiple values). But afaik @normanmaurer plans to plug in the 4.0 headers implementation after this is merged and so that would then keep insertion order.